### DISCUSSION TITLE
Question - Platform url setting

### DISCUSSION THREAD

**Discussion by rakeshdavid** [Budibase/budibase#2360] [General]
Hello,

I am wondering what the Platform url setting do in Organization settings.

I setup NGINX reverse proxy with ssl, and the domain is working great, i am wondering if i have to update and replace http://localhost:1000 to https://budibase.example.com

I have no issues with the app, just wondering if i need to set that up as well. I tried looking up documentation and discussions but could not find any answer. 

<img width="1438" alt="Screen Shot 2021-08-13 at 4 06 41 PM" src="https://user-images.githubusercontent.com/573491/129413564-5c530c42-e08c-496b-9504-a0c74fb2b30b.png">

--
RD

**Comment by mjashanks** [comment]
You should set this to whatever URL your users access budibase on.

Budibase does not automatically know this as you could proxy a custom domain to your IP.

The platform url is then used in things like password reset emails.

**Reply by rakeshdavid** [reply]
Oh, cool, so I should set it up with https://budibase.example.com so that when users reset their password, they get an email from the domain name instead of http://localhost:1000 referenced; in the email?

Btw, thank you for responding quickly; I appreciate it :)

**Reply by mjashanks** [reply]
Yep.

The email doesnt come from that domain, but the reset link within the email links to that domain, eg.

Https://example.budibase.com/resetpassword

**Reply by rakeshdavid** [reply]
Sweet! Thank you @mjashanks
